src/platform/X32/dshot_bitbang.c

```
@@ -0,0 +1,745 @@
+            device->vTable = NULL;
+            dshotMotorCount = 0;
+            bbStatus = DSHOT_BITBANG_STATUS_MOTOR_PIN_CONFLICT;
+            return false;
+        }
+
+        int pinIndex = IO_GPIOPinIdx(io);
+
+        bbMotors[motorIndex].pinIndex = pinIndex;
+        bbMotors[motorIndex].io = io;
+        bbMotors[motorIndex].output = output;
+        bbMotors[motorIndex].iocfg = IO_CONFIG(GPIO_MODE_OUTPUT_PP, GPIO_SLEW_RATE_SLOW, bbPuPdMode,   0x00)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If IO_CONFIG is a macro then please configure it.

(unknownMacro)